client/tests/integration/permissions.rs

```diff
@@ -381,3 +382,50 @@ fn permission_tokens_are_unified() {
         .submit_blocking(allow_alice_to_transfer_rose_2)
         .expect_err("permission tokens are not unified");
 }
+
+#[test]
+fn associated_permission_tokens_removed_on_unregister() {
+    let (_rt, _peer, iroha_client) = <PeerBuilder>::new().with_port(11_240).start_with_runtime();
+    wait_for_genesis_committed(&[iroha_client.clone()], 0);
+
+    let bob_id: AccountId = "bob@wonderland".parse().expect("Valid");
+    let kingdom_id: DomainId = "kingdom".parse().expect("Valid");
```

Comment:
I think we now discourage the use of `expect("Valid")` and prefer just `unwrap()`

Reply:
I was unaware of this, where it's stated?